<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="connectionMenu">
    <property name="title">
     <string>连接</string>
    </property>
    <addaction name="newConnection"/>
    <addaction name="deleteConnectionAction"/>
   </widget>
   <addaction name="connectionMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="newConnection">
   <property name="text">
    <string>新连接</string>
   </property>
  </action>
  <action name="deleteConnectionAction">
   <property name="text">
    <string>删除连接</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
